tests/catalog/test_rest.py

```diff
@@ -450,6 +450,28 @@ def test_list_views_404(rest_mock: Mocker) -> None:
     assert "Namespace does not exist" in str(e.value)
 
 
+def test_view_exists_204(rest_mock: Mocker) -> None:
+    namespace = "examples"
+    view = "some_view"
+    rest_mock.head(
+        f"{TEST_URI}v1/namespaces/{namespace}/views/{view}",
+        status_code=204,
+        request_headers=TEST_HEADERS,
+    )
+    catalog = RestCatalog("rest", uri=TEST_URI, token=TEST_TOKEN)
+    assert catalog.view_exists((namespace, view))
+
+def test_view_exists_404(rest_mock: Mocker) -> None:
+    namespace = "examples"
+    view = "some_view"
+    rest_mock.head(
+        f"{TEST_URI}v1/namespaces/{namespace}/views/{view}",
```

Comment:
Could we add a test case here for a multi-level namespace as well?

Reply:
@sungwy thanks! Is that as simple as creating a test case with:
```python
multilevel_namespace = "core.examples.some_namespace"
view = "some_view"
rest_mock.head(
    f"{TEST_URI}v1/namespaces/{multilevel_namespace}/views/{view}",
    status_code=404,
    request_headers=TEST_HEADERS,
)
```